projects/ad7768-1fmcz/src/ad77681evb.c

```diff
@@ -100,7 +100,7 @@ struct ad77681_init_param ADC_default_init_param = {
 	AD77681_POSITIVE_FS,		// diag_mux_sel
 	false,						// conv_diag_sel
 	AD77681_CONV_16BIT,			// conv_len
-	AD77681_CRC, 				// crc_sel
+	AD77681_NO_CRC, 			// crc_sel
```

Comment:
The description of this commit doesn't contain the reason of this change. Can you provide more details?

Reply:
Will change back to `AD77681_CRC` after #796 is merged to master.